### PR TITLE
Feature/g6 6/raniarayahusercomponent

### PR DESCRIPTION
**Pull request by raniab1903**
separated components, created scoreboard page.

### CODE REVIEW

File: client/src/components/UserScore.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```
Comment (jacquelyn-bradshaw) [inline client/src/components/UserScore.js]: This is your scoreboard page, it just needs renaming to ScoreboardPage.js and putting in the pages folder

File: client/src/pages/ScoreboardPage.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```
Comment (jacquelyn-bradshaw) [inline client/src/pages/ScoreboardPage.js]: this page is not needed

File: client/src/components/UserScore.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+.scoreboard {
+    background-color: #4e00a3;
+    padding: 20px;
+    border-radius: 10px;
+    text-align: center;
+    width: 300px;
+    color: #ffea00;
+  }
+  
+  h2 {
```
Comment (jacquelyn-bradshaw) [inline client/src/components/UserScore.css:11]: this h2 needs to be a class because of how react compiles behind the scenes

File: userscore.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```
Comment (jacquelyn-bradshaw) [inline userscore.js:1]: this component is not needed

**Review by jacquelyn-bradshaw — CHANGES_REQUESTED** [review]
Please can you run `npm lint-fix` in your terminal from the client directory

**Review by oliviaportess — DISMISSED** [review]
Hey guys, there are Button.js and MainHeading.js components that we are using to keep design consistent across all pages, I don't think I can see them in your files at the moment? If you aren't using them yet please could you add them.
Let me know if you need info on what they are/how to use them!

**Comment by soojinhwang** [comment]
Adding to Olivia's comments with the Navbar and the Background screen components
If you have a look at how the other pages have been set up and their file structure (LandingPage is probably the easiest to get) then you'll be able to see how to incorporate the components that have already been made

**Comment by jacquelyn-bradshaw** [comment]
> Hey guys, there are Button.js and MainHeading.js components that we are using to keep design consistent across all pages, I don't think I can see them in your files at the moment? If you aren't using them yet please could you add them. Let me know if you need info on what they are/how to use them!

This change is no longer needed as I have added these components to App.js
https://github.com/oliviaportess/fullstack-6/pull/56

File: userscore.js
Comment (jacquelyn-bradshaw) [inline userscore.js:1]: This whole file is not needed and can be deleted

File: client/src/pages/ScoreBoardPage.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+.scoreboard {
+    background-color: #4e00a3;
+    padding: 20px;
+    border-radius: 10px;
+    text-align: center;
+    width: 300px;
+    color: #ffea00;
+  }
+  
+  h2 {
```
Comment (jacquelyn-bradshaw) [inline client/src/pages/ScoreBoardPage.css]: This will style all h2 elements across the entire app. Please replace h2 tag with a class selector.

**Review by oliviaportess — APPROVED** [review]

**Review by jacquelyn-bradshaw — APPROVED** [review]

**Pull request merged**